libs/classifiers/x86/face_mask.py

```diff
@@ -1,72 +1,72 @@
 import tensorflow as tf
 import numpy as np
-import pathlib
+import wget
+import os
 import time
 from libs.detectors.utils.fps_calculator import convert_infr_time_to_fps
 
 
-def load_model(model_name):
-    """
-    Args:
-        model_name: Download the model based on its name and load the model
-
-    Returns:
-
-    """
-    base_url = 'Not Available'
-    # model_file = model_name + '.tar.gz'
-    # model_dir = tf.keras.utils.get_file(
-    #     fname=model_name,
-    #     origin=base_url + model_file,
-    #     untar=True)
-
-    # model_dir = pathlib.Path(model_dir) / "saved_model"
-    model_dir = 'data/x86/face_mask'  # TODO: load automatically after providing a proper model
-    model = tf.saved_model.load(str(model_dir), None)
-    print('Classifier model signatures: ', list(model.signatures.keys()))
-    model = model.signatures[
-        'predict']  # 'predict' signature was defined during exporting pb model change it if your signiture is someting else
-
-    return model
-
-
 class Classifier:
     """
-    Perform image classification with the given model. The model is a protobuf
-    file which if the classifier can not find it at the path it will download it
+    Perform image classification with the given model. The model is a .h5 file
+    which if the classifier can not find it at the path it will download it
     from neuralet repository automatically.
-
     :param config: Is a ConfigEngine instance which provides necessary parameters.
     """
 
     def __init__(self, config):
         self.config = config
-        self.model_name = self.config.get_section_dict('Classifier')['Name']
-        self.classifier_model = load_model(self.model_name)
+        self.model_path = self.config.get_section_dict('Classifier')['ModelPath']
```

Comment:
I think we should handle this the same way for all platforms, for example EdgeTPU/Coral don't use this config I think